log/fatal.go

```diff
@@ -0,0 +1,64 @@
+package log
+
+import (
+	"fmt"
+)
+
+// Common errors that can happen on node startup.
+var (
+	ErrMalformedConfig    = newFatalError("ERR_MALFORMED_CONFIG", "config file is malformed: %v")
+	ErrBadFlags           = newFatalError("ERR_BAD_FLAGS", "bad CLI flags: %v")
+	ErrTortoiseHareParams = newFatalError("ERR_TORTOISE_HARE_PARAMS", "incompatible tortoise hare params: %v")
+	ErrEnsureDataDir      = newFatalError("ERR_ENSURE_DATA_DIR", "could not open/create data dir %v: %v")
```

Comment:
templates like this `could not open/create data dir %v: %v` should be defined where the error is created, not in some random module